### DISCUSSION TITLE
add AutoMigrate

### DISCUSSION THREAD

**Discussion by Baiyuetribe** [SeaQL/sea-orm#1985] [Ideas]
## Motivation
Creating, updating, and modifying database tables is a frequent task, and it would be beneficial to have an automated process similar to Go's capabilities.

## Proposed Solutions

like go
```go
 db, err := gorm.Open(sqlite.Open("test.db"), &gorm.Config{})
  if err != nil {
    panic("failed to connect database")
  }

  // Migrate the schema
  db.AutoMigrate(&Product{})

```
https://github.com/go-gorm/gorm/blob/3207ad6033aad5e76c6c9d578ef663032765e484/migrator/migrator.go#L114C24-L114C24

**Comment by Millione** [comment]
Is there any plan to support this?

**Reply by tyt2y3** [reply]
Yeah, hopefully we'll implement it this year.
